parkhangs-webapp/src/components/Marker.js

```diff
@@ -28,19 +28,21 @@ class Marker extends Component {
     render() {
         return (
             <div>
-                <img alt="map-marker" src={MarkerIcon}/>
                 <div className="marker" onClick={() => {
                     this.handleSelect(this.props.park)
                 }}
                 >
+                  <img alt="map-marker" src={MarkerIcon}/>   
                     {/*TODO: should we display the name of the park*/}
-                    {this.props.park.name}
+                    {/* {this.props.park.name} */}
```

Comment:
could you just delete this line? I think we're pretty sure we don't want the park name there anymore

Reply:
Kk